srcpkgs/bastet/template

```diff
@@ -0,0 +1,29 @@
+# Template file for 'bastet'
+pkgname=bastet
+version=0.43.2
+revision=1
+archs="~*-musl"
```

Comment:
Can you privide the reason musl is disabled?

Reply:
Building with musl encountered some errors, I'll try again and see if I can work out a patch for it

Reply:
I have re-enabled it now, turns out I had something set up wrong and it builds with musl now, sorry

Reply:
I've managed to patch it to work with musl and built for x86_64-musl